src/transformers/models/llama/modeling_llama.py

```diff
@@ -630,8 +630,6 @@ def custom_forward(*inputs):
 
 
 class LlamaForCausalLM(LlamaPreTrainedModel):
-    _keys_to_ignore_on_load_missing = [r"lm_head.weight"]
```

Comment:
This has to be removed since LLama doesnt' share the embedding and the output.

@ArthurZucker for final confirmation ?